templates/todo/common/infra/bicep/app/api-container-app.bicep

```diff
@@ -18,6 +18,8 @@ module api '../../../../../common/infra/bicep/core/host/container-app.bicep' = {
     location: location
     containerAppsEnvironmentName: containerAppsEnvironmentName
     containerRegistryName: containerRegistryName
+    containerCpuCoreCount: '1.0'
+    containerMemory: '2.0Gi'
```

Comment:
This bumps up the default cores for ACA from 0.5 to 1.0, and memory usage from 1GB to 2GB, which makes it more inline with the AppService plan we use.